### DISCUSSION TITLE
What firewall rules are required by the management service?

### DISCUSSION THREAD

**Discussion by ser** [netbirdio/netbird#684] [Q&A / Support]
What firewall rules are required by the management services? (+ signal, turn, etc. - all required for self-hosting)

**Comment by mlsmaycon** [comment]
Hello @ser the ports are documented in the Requirements section of our self-hosted guide: https://netbird.io/docs/getting-started/self-hosting#requirements

**Comment by ser** [comment]
Hi, I'm asking as that information is not complete. There is nothing about outgoing traffic, which is substantial as far as I can see. Management services are trying to contact some servers belonging to your company. Is it possible to clearly identify IP ranges of these devices and port ranges? What is the purpose of these devices and why they can't be replaced by self hosted services?
